{"name": "test_auth_biometric", "status": "failed", "statusDetails": {"message": "AssertionError: Expected 'Failed', but got 'None'\nassert None == 'Failed'", "trace": "self = <TestCases.test_biometric_auth.Test_Bio_Auth object at 0x000002DDFD060C90>\n\n    def test_auth_biometric(self):\n        home = HomeScreen(self.driver)\n        home.goToAndroidBioMetric().clickOnPassCallBack()\n        actualMessage = home.goToAndroidBioMetric().getStatusMessage()\n    \n        expectedMessage = 'Failed'\n    \n>       assert actualMessage == expectedMessage, f\"Expected '{expectedMessage}', but got '{actualMessage}'\"\nE       AssertionError: Expected 'Failed', but got 'None'\nE       assert None == 'Failed'\n\ntest_biometric_auth.py:14: AssertionError"}, "attachments": [{"name": "log", "source": "ecb65304-3300-48c1-bef9-6e2dff62dc8d-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "db252504-7aa9-4025-a9fd-abfdfdc22f66-attachment.txt", "type": "text/plain"}], "start": 1754398065267, "stop": 1754398067313, "uuid": "43b5de49-5249-4ec5-8218-b891f561407b", "historyId": "476a68ff226e5853fa3aa36fa902fed3", "testCaseId": "476a68ff226e5853fa3aa36fa902fed3", "fullName": "test_biometric_auth.Test_Bio_Auth#test_auth_biometric", "labels": [{"name": "suite", "value": "test_biometric_auth"}, {"name": "subSuite", "value": "Test_Bio_Auth"}, {"name": "host", "value": "Abhishek"}, {"name": "thread", "value": "9020-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_biometric_auth"}], "titlePath": ["test_biometric_auth.py", "Test_Bio_Auth"]}